torchtext/data/functional.py

```diff
@@ -16,7 +16,7 @@
 def generate_sp_model(filename, vocab_size=20000,
                       model_type="unigram",
                       model_prefix='m_user'):
-    """Train a SentencePiece tokenizer.
+    r"""Train a SentencePiece tokenizer.
```

Comment:
nit: This string change could have gone in a separate PR :)

Reply:
I feel it's a very small change so I just made it here :).